boards/arm/particle_argon_nrf52/particle_argon_nrf52.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2018 Matthias Boesl
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "particle argon nrf52";
+	compatible = "particle,particle_argon", "nordic,nrf52840-qiaa",
+		     "nordic,nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		user_led: led_0 {
+			gpios = <&gpio1 12 0>;
+			label = "User LED";
+		};
+		status_green: led_1 {
+				gpios = <&gpio1 14 0>;
+				label = "Green LED";
+		};
+		status_blue: led_2 {
+				gpios = <&gpio1 15 0>;
+				label = "Blue LED";
+		};
+		status_red: led_3 {
+				gpios = <&gpio1 13 0>;
+				label = "Red LED";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		mode_button: button_0 {
+			gpios = <&gpio0 11 GPIO_PUD_PULL_UP>;
+			label = "Mode Button";
+		};
+
+		reset_button: button_1 {
+			gpios = <&gpio0 18 GPIO_PUD_PULL_UP>;
+			label = "Reset Button";
+		};
+	};
+
+	aliases {
+		led0 = &user_led;
+		led1 = &status_green;
+		led2 = &status_blue;
+		led3 = &status_red;
+		sw0 = &mode_button;
+		sw1 = &reset_button;
+	};
+};
+
+&gpiote {
+	status ="ok";
+};
+
+&gpio0 {
+	status ="ok";
+};
+
+&gpio1 {
+	status ="ok";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "ok";
+	tx-pin = <6>;
+	rx-pin = <8>;
+	rts-pin = <2>;
+	cts-pin = <3>;
```

Comment:
There is some discrepancy here. According to schematics one UART is connected to ESP32 and uses the following pins:
- tx: 37 (p1.05)
- rx: 36 (p1.04)
- rts: 39 (p1.07)
- cts: 38 (p1.06)

and the other one is for external usage and uses only two pins:
- tx: 6 (p0.6)
- rx: 8 (p0.8)